dlt/destinations/insert_job_client.py

```diff
@@ -72,6 +72,33 @@ def _insert(self, qualified_table_name: str, file_path: str) -> Iterator[List[st
                         else:
                             # Replace the , with ;
                             insert_sql.append("".join(chunk).strip()[:-1] + ";\n")
+                elif max_rows >= 1000:
```

Comment:
i think you need to also test for max_rows is not None here, otherwise the gte operator will fail on none.